scripts/lib/helm.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+add_helm_repo() {
+   if ! should_execute add_helm_repo; then
+     return 1
+   fi
+
+   # add the helm repository containing charts for starting AWS service controllers
+   if ! helm repo add "$HELM_LOCAL_REPO_NAME" "$HELM_REPO_SOURCE" > /dev/null 2>&1; then
+     echo "Unable to add local helm repo from '$HELM_REPO_SOURCE'"
+     TEST_PASS=1
+     return 1
+   fi
+
+  #list the charts in the the local repo
+  echo "Validating the presence of '$HELM_REPO_CHART_NAME' in local repo '$HELM_LOCAL_REPO_NAME' "
+  helm_repo_output_lines=$(helm search repo "$HELM_LOCAL_REPO_NAME" | grep "$HELM_REPO_CHART_NAME" | wc -l)
+  if [[ "$helm_repo_output_lines" -gt 0 ]]; then
+    echo "'$HELM_REPO_CHART_NAME' chart is present in local helm repo '$HELM_LOCAL_REPO_NAME'"
+  else
+    echo "'$HELM_REPO_CHART_NAME' chart is NOT present in local helm repo '$HELM_LOCAL_REPO_NAME'."
+    TEST_PASS=1
+  fi
+}
+
+install_helm() {
+  # install helm in /tmp directory
+  pushd /tmp
+  # clone the source
+  git clone https://github.com/helm/helm.git
```

Comment:
Maybe it's safer to clone a stable release instead of the master branch?

Reply:
Very good point. Although they do not mention any specific branch/tag as stable.

I can pick any stable(pre-released) branch ATM, but do you have thoughts on how to keep it up to date, and avoid the risk of it being deprecated. 
I am guessing it has to be a periodical up-keeping item.

Reply:
Good point 👍 , I agree we can use a pre-released version for now and it's better if can keep it up to date.
I can think of two solutions:
- We can inject a `$HELM_RELEASE` environment variable using CircleCI console and try to "manually" keep it up to date.
- clone the master branch and try to checkout the latest pre-release tag:
```bash
git fetch --tags
git checkout $(git tag -l | tail -1)
# this can also work to fetch latest remote tag:
# git ls-remote --tags --sort=committerdate | grep -o 'v.*' | sort -r | head -1
```